rover/src/owr_messages/msg/science.msg

```diff
@@ -0,0 +1,9 @@
+uint32 temperature
+uint32 humidity
+float32 mag_x
+float32 mag_y
+float32 mag_z
```

Comment:
are we still using the magnometer lol, I thought we decided to leave it

Reply:
We are using it